devcontainer: Missing xelatex dependency to build PDF documentation

<!-- Please insert your suggestion or idea below this line -->

The following error occurred when I attempted to run the `make pdf` command:

```text
$ make pdf

...stripped...

build succeeded, 7 warnings.

The LaTeX files are in _build/latex.
Run 'make' in that directory to run these through (pdf)latex
(use `make latexpdf' here to do that automatically).
Running LaTeX files through pdflatex...
buf_size=10000000 make -C _build/latex all-pdf
make[2]: Entering directory '/workspaces/Nextcloud documentation/admin_manual/_build/latex'
latexmk -pdf -dvi- -ps-  'Nextcloud_Server_Administration_Manual.tex'
Rc files read:
  /etc/LatexMk
  latexmkrc
Latexmk: This is Latexmk, John Collins, 11 Dec. 2024. Version 4.86.
Latexmk: applying rule 'pdflatex'...
Rule 'pdflatex':  Reasons for rerun
Changed files or newly in use/created:
  Nextcloud_Server_Administration_Manual.aux
  Nextcloud_Server_Administration_Manual.tex
Category 'changed_user':
  Nextcloud_Server_Administration_Manual.tex

------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex   -recorder  "Nextcloud_Server_Administration_Manual.tex"'
------------
sh: 1: xelatex: not found
Latexmk: No log file was found, neither the expected one, 'Nextcloud_Server_Administration_Manual.log', nor one in '.'.
Latexmk: fls file doesn't appear to have been made.
Latexmk: Getting log file 'Nextcloud_Server_Administration_Manual.log'
Latexmk: Couldn't read log file 'Nextcloud_Server_Administration_Manual.log':
  No such file or directory
Latexmk: Errors, so I did not complete making targets
Collected error summary (may duplicate other messages):
  pdflatex: Run of rule 'pdflatex' gave a non-zero error code
----------------------
This message may duplicate earlier message.
Latexmk: Failure in processing file 'Nextcloud_Server_Administration_Manual.tex':
   *LaTeX didn't generate the expected log file 'Nextcloud_Server_Administration_Manual.log'
----------------------

Latexmk: Sometimes, the -f option can be used to get latexmk
  to try to force complete processing.
  But normally, you will need to correct the file(s) that caused the
  error, and then rerun latexmk.
  In some cases, it is best to clean out generated files before rerunning
  latexmk after you've corrected the files.
make[2]: *** [Makefile:29: Nextcloud_Server_Administration_Manual.pdf] Error 12
make[2]: Leaving directory '/workspaces/Nextcloud documentation/admin_manual/_build/latex'
make[1]: *** [Makefile:118: latexpdf] Error 2
make[1]: Leaving directory '/workspaces/Nextcloud documentation/admin_manual'
make: *** [Makefile:22: admin-manual-pdf] Error 2
```

it appears that the dependency is not installed in the environment.

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -6,7 +6,7 @@ USER root
 RUN export DEBIAN_FRONTEND=noninteractive \
     && apt-get update \
     && apt-get -y install --no-install-recommends \
-        curl gnupg2 git nano make graphviz imagemagick inkscape sass unzip wget php-cli npm latexmk texlive-latex-extra tex-gyre \
+        curl gnupg2 git nano make graphviz imagemagick inkscape sass unzip wget php-cli npm latexmk texlive-latex-extra tex-gyre texlive-xetex \
     && wget https://getcomposer.org/installer -qO /tmp/composer-setup.php \
     && php /tmp/composer-setup.php \
     && mv composer.phar /usr/local/bin/composer \

